### ISSUE TITLE
Themes: "Upload Theme" instead of "Install Theme"

### ISSUE THREAD

**Issue by DanielSmithMichigan** [Automattic/wp-calypso#53599]
We should use the wording "Upload Theme" instead of "Install Theme" for the button shown below.

<img src="https://user-images.githubusercontent.com/10274366/121567806-fd0ec300-c9ec-11eb-823e-5463ab9089b9.png" width="350">

### Why?

- It's more in line with Core.
- "install theme" is close in meaning to "activate theme" and users could get these two actions confused.
- "upload theme" is dissimilar in meaning to "activate theme," so it's less open to misinterpretation.
- Additionally, a user who has a theme zip file naturally knows the file needs to be uploaded to take effect. These users are probably already looking for a button which says "upload" since that is what they intend to do with the zip file.

Additional thoughts and reasoning here: pd2qbF-6Y-p2

**Comment by Aurorum** [comment]
PR for this issue in #53622. :)

**Comment by jsnajdr** [comment]
@onuro @DanielSmithMichigan We used to call the button "Upload Theme" in the past, but then @jaclync changed it to "Install Theme" in #36136 in September 2019. The rationale was consistency with similar "install/upload plugin" actions in the Plugins section.

I don't disagree with changing it back, just wanted to share some prior art and context about this question.

**Comment by ianstewart** [comment]
@DanielSmithMichigan looks like core uses "upload" for plugins as well.

![image](https://user-images.githubusercontent.com/545779/121939312-833f4800-cd12-11eb-95d2-cacce2213b9a.png)

Whereas we use install for plugins in Calypso.

![image](https://user-images.githubusercontent.com/545779/121939382-97834500-cd12-11eb-918a-9cef93cfe55b.png)

When we close this issue we should create a new one to use "Upload" for plugins as well, like so:

![image](https://user-images.githubusercontent.com/545779/121939517-b84b9a80-cd12-11eb-979a-eba84f98dad4.png)

**Comment by ianstewart** [comment]
We should take a look at where "install" appears as the verb in further screens like this eligibility warning that uses "install" in the headercake menu and the eligibility warning itself.

![image](https://user-images.githubusercontent.com/545779/121939678-e0d39480-cd12-11eb-9513-4644df5b7d9b.png)

**Comment by kwight** [comment]
I don't have a strong feeling either way, but I do feel odd about introducing a new inconsistency with the plugin screens language (particularly when we've already "fixed" it once, as @jsnajdr pointed out). I'd prefer we drop this with no action, but I'm happy to leave the final decision to you @ianstewart .

**Comment by ianstewart** [comment]
> I do feel odd about introducing a new inconsistency with the plugin screens language

Yeah, agreed. Though I'll note there's an additional baked-in inconsistency. After going Atomic the "install" button turns into an "upload" button. That said, after looking at the eligibility warnings, and noting it's not a bug, I think we should correct it everywhere. Going to move this one to "Icebox" as it's not a bug and suggest that we cover all the cases in the flows at once.
